mantidproject/mantid
# ILL Powder Diffraction Detector Scan Reduction

Add a workflow algorithm for powder diffraction reduction at the ILL with detector scans. Should support D2B and D20, although we are lacking data to test D20 with at the moment.

**To test:**

Download the attached test files and add the data directory to Mantid. Be sure to set the facility to ILL. Try running the workflow algorithm with the various options. Check documentation.

[D2B_Example.zip](https://github.com/mantidproject/mantid/files/1625765/D2B_Example.zip)

Fixes #21456.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
